plasmapy/simulation/abstractions.py

```diff
@@ -54,3 +54,142 @@ class AbstractTimeDependentSimulation(AbstractSimulation):
     """
 
     pass
+
+
+class AbstractNormalizations(ABC):
+    """
+    An abstract base class to represent the normalizations of systems of
+    equations describing plasmas.
+
+    Warnings
+    --------
+    This interface is unstable and is subject to change.
```

Comment:
Might be better to use the `reST` [warning admonition](https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions) here so this does not get confused with `numpydoc`'s warning documentation.  This should work, but I've never done it like this before.

```suggestion
    .. warning::
       This interface is unstable and is subject to change.
```

Reply:
Good idea!  One thing I read in the numpydoc docs is that they have sections named "Warns" which is for warnings that are issued, and "Warnings" which are intended for readers.

Reply:
I did not know that.  I wonder if it renders like  the reST admonition. 🤔  I went to check the old RTD builds, but it wasn't rendered because `AbstractNormalizations` was not included in the `__all__`.